DllImportGenerator/DllImportGenerator/DllImportStub.cs

```diff
@@ -240,5 +241,31 @@ public class GeneratedDllImportData
                 AdditionalAttributes = additionalAttrs.ToArray(),
             };
         }
+
+        private static bool IsSkipLocalsInit(IMethodSymbol method)
```

Comment:
This checks really seems expensive to be honest. Is there a way we can optimize for the module level case?

Reply:
I'll go precalculate the module case.